src/App.tsx

```diff
@@ -1,27 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const loadAll = () => (
```

Comment:
Server has only 1 endpoint returning all the goods, so you should do all the preparations in corresponding methods in /api/goods. So, this functions should be in /api/goods. You can use  import { getAll, get5First, getRed } from './api/goods';
 or
import * as goodsAPI from './api/goods';